ca_gazebo/launch/spawn_virtual_wall.launch

```diff
@@ -0,0 +1,11 @@
+<launch>
```

Comment:
Put this launchfile inside `ca_gazebo/launch/include/spawn_virtual_wall.launch`.